### ISSUE TITLE
santa_clara_valley

### ISSUE THREAD

**Issue by qjhart** [UCDavisLibrary/ava#188]
# AVA: Santa Clara Valley (santa_clara_valley)

name | value
--- | ---
ava_id | santa_clara_valley
cfr_index | 9.126
revision | [T.D. ATF-286, 54 FR 12606, Mar. 28, 1989]
state | CA
county | Alameda|San Benito|Santa Clara
within | Central Coast\|San Francisco Bay
contains | Pacheco Pass\|San Ysidro District

[src]: https://www.ecfr.gov/cgi-bin/retrieveECFR?gp=&SID=371db32ecca6629af6dccad2a39d7833&mc=true&n=sp27.1.9.c
## Approved Maps [src]

(b) Approved Maps. The appropriate maps for determining the boundaries of the “Santa Clara Valley” viticultural area are 25 U.S.G.S. Quadrangle (7.5 Minute Series) maps. They are titled:

(1) Calaveras Reservoir, Calif., 1961 (photorevised 1980);

(2) Castle Rock Ridge, Calif., 1955 (photorevised 1968), photoinspected 1973;

(3) Chittenden, Calif., 1955 (photorevised 1980);

(4) Cupertino, Calif., 1961 (photorevised 1980);

(5) Gilroy, Calif., 1955 (photorevised 1981);

(6) Gilroy Hot Springs, Calif., 1955 (photorevised 1971), photoinspected 1973;

(7) Lick Observatory, Calif., 1955 (photorevised 1968), photoinspected 1973;

(8) Loma Prieta, Calif., 1955 (photorevised 1968);

(9) Los Gatos, Calif., 1953 (photorevised 1980);

(10) Milpitas, Calif., 1961 (photorevised 1980);

(11) Mindego Hill, Calif., 1961 (photorevised 1980);

(12) Morgan Hill, Calif., 1955 (photorevised 1980);

(13) Mt. Madonna, Calif., 1955 (photorevised 1980);

(14) Mt. Sizer, Calif., 1955 (photorevised 1971), photoinspected 1978;

(15) Mountain View, Calif., 1961 (photorevised 1981);

(16) Newark, Calif., 1959 (photorevised 1980);

(17) Niles, Calif., 1961 (photorevised 1980);

(18) Pacheco Peak, Calif., 1955 (photorevised 1971);

(19) Palo Alto, Calif., 1961 (photorevised 1973);

(20) San Felipe, Calif., 1955 (photorevised 1971);

(21) San Jose East, Calif., 1961 (photorevised 1980);

(22) San Jose West, Calif., 1961 (photorevised 1980);

(23) Santa Teresa Hills, Calif., 1953 (photorevised 1980);

(24) Three Sisters, Calif., 1954 (photorevised 1980);

(25) Watsonville East, Calif., 1955 (photorevised 1980);

## Boundary [src]

(c) The boundaries of the proposed Santa Clara Valley viticultural area are as follows:

(1) The beginning point is at the junction of Elephant Head Creek and Pacheco Creek (approx. .75 mile southwest of the Pacheco Ranger Station) on the Pacheco Peak, Calif. U.S.G.S. map.

(2) From the beginning point the boundary moves in a northerly direction up Elephant Head Creek approx. 1.2 miles until it intersects the 600 foot elevation contour line;

(3) Then it meanders in a northwesterly direction along the 600 foot contour line approx. 55 miles until it intersects Vargas Road in the northwest portion of Sec. 25, T4S/RlW on the Niles, Calif. U.S.G.S. map;

(4) Then it travels in a northwesterly direction approx. .6 mile to the intersection of Morrison Canyon Road in the eastern portion of Sec. 23, T4S/RlW;

(5) Then it follows Morrison Canyon Road west approx. 1.5 miles to Mission Boulevard (Highway 238) at Sec. 22, T4S/RlW;

(6) Then it moves northwest on Mission Boulevard (Highway 238) approx. .6 mile to the intersection of Mowry Avenue just past the Sanatorium at Sec. 22, T4S/RlW;

(7) It then goes in a southwesterly direction on Mowry Avenue approx. 3.6 miles to the intersection of Nimitz Freeway (Highway 880) (depicted on the map as Route 17) at Sec. 5, T5S/RlW, on the Newark, Calif. U.S.G.S. map;

(8) It then moves along the Nimitz Freeway (Highway 880) in a southeasterly direction for approx. 9 miles to the intersection of Calaveras Boulevard (Highway 237) at Milpitas on the Milpitas, Calif. U.S.G.S. map;

(9) Then it follows Highway 237 in a westerly direction approx. 7.2 miles to intersection of Bay Shore Freeway (Highway 101) at Moffett Field on the Mt. View, Calif. U.S.G.S. map;

(10) Then in a northwest direction follow Bay Shore Freeway (Highway 101) for approx. 6.5 miles to the intersection of the San Francisquito Creek (Santa Clara County/San Mateo County boundary) at Palo Alto T5S/R2W, on the Palo Alto, Calif. U.S.G.S. map;

(11) Then it heads west on San Francisquito Creek (Santa Clara County/San Mateo County boundary) approx. 7 miles until it converges with Los Trancos Creek (Santa Clara County/San Mateo County boundary) near Bench Mark 172, approx. 100 feet east of Alpine Road;

(12) It travels south approx. 4 miles along Los Trancos Creek (Santa Clara County/San Mateo County boundary) until it intersects the 600 foot elevation contour line at El Corte De Madera, approx. .5 mile north of Trancos Woods on the Mindego Hill, Calif. U.S.G.S. map;

(13) It moves along the 600 foot elevation contour line in a southeasterly direction approx. 10 miles to Regnart Road at Regnart Creek on the Cupertino, Calif. U.S.G.S. map;

(14) It goes northeast along Regnart Road, approx. .7 mile to the 400 foot elevation contour line (.3 mile southwest of Regnart School);

(15) It travels along the 400 foot elevation contour line southeast approx. 1.4 miles to the north section line of Section 36, T7S/R2W at Blue Hills, CA;

(16) The boundary goes east on the section line approx. .4 mile to Saratoga Sunnyvale Road (Highway 85);

(17) It travels south on Saratoga Sunnyvale Road (Highway 85) approx. 1 mile to the south section line of Section 36, T7/8S R2W;

(18) Then it goes west on the section line approx. .75 mile to the first intersection of the 600 foot elevation contour line;

(19) It follows the 600 foot elevation contour line southeast approx. .75 mile to Pierce Road south of Calabazas Creek;

(20) It then travels south on Pierce Road approx. .4 mile to the first intersection of the 800 foot elevation contour line;

(21) Then it runs southeast approx. 28 miles on the 800 foot elevation contour line to the east section line of Sec. 25, T10S/R2E/R3E approx. .5 mile north of Little Arthur Creek on the Mt. Madonna, Calif. U.S.G.S. map;

(22) Then it goes south on the section line approx. .5 mile to the 800 foot elevation contour line approx. .2 mile south of Little Arthur Creek;

(23) Then it goes southeast along the 800 foot elevation contour line approx. 2.7 miles to Hecker Pass Road (Highway 152) approx. 1.25 miles east of Hecker Pass on the Watsonville East, Calif. U.S.G.S. map;

(24) The boundary goes northeast on Hecker Pass Road (Highway 152) approx. .75 mile to the intersection of the 600 foot elevation contour line just west of Bodfish Creek;

(25) It travels southeast along the 600 foot elevation contour line approx. 7.3 miles to the first intersection of the western section line of Sec. 30, T11S/R3E/R4E on the Chittenden, Calif. U.S.G.S. map;

(26) Then it follows south along the section line approx. 1.9 miles to the south township line at Sec. 31, T11S/T12S, R3E/R4E;

(27) It moves in an easterly direction along the township line approx. 12.4 miles to the intersection of T11S/T12S and R5E/R6E on the Three Sisters, Calif. U.S.G.S. map;

(28) Then it goes north along R5E/R6E range line approx. 5.3 miles to Pacheco Creek on the Pacheco Creek, Calif. U.S.G.S. map;

(29) Then it moves northeast along Pacheco Creek approx. .5 mile to Elephant Head Creek at the point of beginning.

[T.D. ATF-286, 54 FR 12606, Mar. 28, 1989]

**Comment by MicheleTobias** [comment]
I'm working on this one

**Comment by SoominHur** [comment]
Reviewed.
- Edit cfr_author name in the attribute table
- Edit petitioner name in the attribute table
- For review, I could not find (6) "Gilroy Hot Springs, CA" 1955, photorevised 1971, photoinspected 1973. Instead I used "Gilory Hot Springs, CA" 1955, photorevised 1971, which is the same edition that Michele recorded as a used maps in the attribute table. 
-  For review, I could not find (24) "Three Sisters, CA" 1954, photorevised 1980. Instead I used "Three Sisters, CA" 1954, photorevised 1971, photoinspected 1978.
- Edit (25) because it suppose to meander "along the 600 foot elevation contour line ... to the first intersection of the western section line of Sec. 30, T11S/R3E/R4E on the Chittenden, Calif. U.S.G.S. map". But the line kept meandering the 600' contour line to the second intersection of the western section line of Section 30 (T11S/R3E/R4E).
